src/client/components/pages/HomePage/Home.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function Home() {
+  return (
+    <div className="about-page">
+      <h1 className="about-page-h1"> Unforgettable culinary experiences.</h1>
+      <p>Join a fantastic culinary experiences led by passionate chefs. </p>
+      <img
+          src="https://www.stephenconroy.com/images/GALLERYVIEW_575_MissSaigon_Hero_web.jpg"
+          alt="Meal Picture"
+          width="1150px"
+          height="500px"
+          width="100%"
+          height="49%"
```

Comment:
You have duplicate widths and heights here.

Reply:
oh! i kept them both for testing purposes and forgot to remove. Done now :)